thentos-core/src/Thentos/Util.hs

```diff
@@ -86,16 +83,6 @@ readsPrecEnumBoundedShow _ s = f [minBound..]
         s' = show x
 
 
--- | Path concatenation for avoiding double slashes in paths.  One
```

Comment:
Why was this moved?  And why the trivial extra sub-functions?  I would prefer to undo this; if you disagree, the sub-functions should at least be marked with the INLINE pragma (even though that may be happening automatically).